types/lodash/lodash-tests.ts

```diff
@@ -13774,16 +13774,16 @@ namespace TestOverEvery {
 // _.overSome
 namespace TestOverSome {
     {
-        let result: (...args: any[]) => boolean;
+        let result: (...args: number[]) => boolean;
 
-        result = _.overSome(() => true);
-        result = _.overSome(() => true, () => true);
-        result = _.overSome([() => true]);
-        result = _.overSome([() => true], [() => true]);
+        result = _.overSome<number>((n: number) => true);
```

Comment:
I'd be inclined to remove the explicit `<number>` constraint and test that typescript constrains it implicitly.

Reply:
Will do, thanks!

Do you know if it's also possible to test that TS should error if the function parameter doesn't match the expected generic? This would be to check that the generic is actually flowing through and not just widening to `any`.

Reply:
Not that I know of. I would love that if it was possible though!